tools/universe_builder.py

```diff
@@ -1,237 +1,62 @@
-            logger.info('  {{%s}} => %s' % (key, template_mapping[key]))
-        logger.info('Resulting diff:')
-        logger.info('\n'.join(difflib.ndiff(
-            orig_content.split('\n'), new_content.split('\n'))))
-        return new_content
-
-    def _generate_packages_dict(self, package_files):
-        package_json = json.loads(
-            package_files[_package_json_filename], object_pairs_hook=collections.OrderedDict)
-        package_json['releaseVersion'] = 0
-
-        command_json = package_files.get(_command_json_filename)
-        if command_json is not None:
-            package_json['command'] = json.loads(
-                command_json, object_pairs_hook=collections.OrderedDict)
-
-        config_json = package_files.get(_config_json_filename)
-        if config_json is not None:
-            package_json['config'] = json.loads(
-                config_json, object_pairs_hook=collections.OrderedDict)
-
-        marathon_json = package_files.get(_marathon_json_filename)
-        if marathon_json is not None:
-            package_json['marathon'] = {
-                'v2AppMustacheTemplate': base64.standard_b64encode(bytearray(marathon_json, 'utf-8')).decode()}
-
-        resource_json = package_files.get(_resource_json_filename)
-        if resource_json is not None:
-            package_json['resource'] = json.loads(
-                resource_json, object_pairs_hook=collections.OrderedDict)
-
-        return {'packages': [package_json]}
-
-    def build_package(self):
-        '''builds a stub universe json package and returns its location on disk'''
-
-        # read files into memory and apply templating to files:
-        updated_package_files = {}
-        for filename, content in self._iterate_package_files():
-            updated_package_files[filename] = self._apply_templating_to_file(
-                filename, content)
-        scratchdir = tempfile.mkdtemp(prefix='stub-universe-tmp')
-        jsonpath = os.path.join(
-            scratchdir, 'stub-universe-{}.json'.format(self._pkg_name))
-        jsonfile = open(jsonpath, 'w')
-        jsonfile.write(json.dumps(self._generate_packages_dict(
-            updated_package_files), indent=2))
-        jsonfile.flush()
-        jsonfile.close()
-        return jsonpath
-
-
-def print_help(argv):
-    logger.info(
-        'Syntax: {} <package-name> <package-version> <template-package-dir> <artifact-base-path> [artifact files ...]'.format(argv[0]))
-    logger.info(
-        '  Example: $ {} kafka 1.2.3-4.5.6 /path/to/template/jsons/ https://example.com/path/to/kafka-artifacts /path/to/artifact1.zip /path/to/artifact2.zip /path/to/artifact3.zip'.format(argv[0]))
-    logger.info(
-        'In addition, environment variables named \'TEMPLATE_SOME_PARAMETER\' will be inserted against the provided package template (with params of the form \'{{some-parameter}}\')')
+EPILOG_STRING = "In addition, environment variables named 'TEMPLATE_SOME_PARAMETER' " \
```

Comment:
`EPILOGUE`?

Reply:
Was basing the name on the fact that `argparse` refers to this as the `epilog`: https://docs.python.org/3/library/argparse.html#epilog, but will change it.